lib/topological_inventory/ansible_tower/operations/ansible_tower_client.rb

```diff
@@ -0,0 +1,128 @@
+require "topological_inventory/ansible_tower/logging"
+require "topological_inventory/ansible_tower/connection_manager"
+require "topological_inventory/providers/common/mixins/sources_api"
+require "topological_inventory/providers/common/mixins/x_rh_headers"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      class AnsibleTowerClient
```

Comment:
This is not new class, but moved from TopologicalInventory::AnsibleTower::Operations::Core::AnsibleTowerClient. I don't know why it's not compared to original file (deleted in this PR)